_docs/setup/install-kubernetes.md

```diff
@@ -73,59 +71,29 @@ default namespace. They can be modified for deployment in a different namespace.
-   ```
-   You need to add the following: (replace the name with your own)
-   ```
-   kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@example.org
-   ```
-   
-   * If the command displays only 'alpha' version, please apply istio-rbac-alpha.yaml configuration:
-   
-   *(Note: If you deploy Istio in another namespace than the `default` namespace, replace the `namespace: default` line 
-      in all ClusterRoleBinding resources with the actual namespace.)*
-      
-   ```bash
-   kubectl apply -f install/kubernetes/istio-rbac-alpha.yaml
-   ```
-
-1. Install Istio's core components .
-   There are two mutually exclusive options at this stage:
-
-    * Install Istio without enabling [Istio Auth]({{home}}/docs/concepts/network-and-auth/auth.html) feature:
+    * Install Istio without enabling [authentication]({{home}}/docs/concepts/network-and-auth/auth.html) between sidecars with [mutual TLS authentication](https://en.wikipedia.org/wiki/Mutual_authentication).
+    We recommend this option for clusters with existing applications, to allow new applications to communicate with the old applications,
+    and for clusters that use [liveliness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), headless services, or statefulsets.
 
       ```bash
       kubectl apply -f install/kubernetes/istio.yaml
       ```
-   
-      This command will install Pilot, Mixer, Ingress-Controller, Egress-Controller core components.
 
-   * Install Istio and enable [Istio Auth]({{home}}/docs/concepts/network-and-auth/auth.html) feature
-   (This deploys a CA in the namespace and enables
-   [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication) between the services):
+   * Install Istio and enable [authentication]({{home}}/docs/concepts/network-and-auth/auth.html) between sidecars with [mutual TLS authentication](https://en.wikipedia.org/wiki/Mutual_authentication):
 
      ```bash
      kubectl apply -f install/kubernetes/istio-auth.yaml
      ```
-     This command will install Pilot, Mixer, Ingress-Controller, and Egress-Controller, and the Istio CA (Certificate Authority).
+
+   This creates the `istio-system` namespace, all necessary RBAC permissions, and deploys Pilot, Mixer, Istio-Ingress, Istio-Egress, and Istio-CA (Certificate Authority).
+
+1. *Optional:* If you cluster has Kubernetes alpha features enabled, and you wish to enable transparent injection of sidecar, please install the Istio-Initializer:
```

Comment:
can we please be explicit about which alpha feature and point a link to this?  https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature  

ideally, we should also show user how to check for this, e.g. via `kubectl api-versions` and check if admissionregistration.k8s.io/v1alpha1 is there.